src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldMediaProperties.class.php

```diff
@@ -11,6 +11,9 @@
 
 use ChameleonSystem\CoreBundle\ServiceLocator;
 
+/**
+ * @deprecated since 6.3.0 - it is a very old mangling of a "number"
```

Comment:
```suggestion
 * @deprecated since 6.3.0 - no longer used
```

Reply:
I like my version. Maybe we can combine them? :-)

Reply:
Not quite sure - your version is mostly judging the oddity of the class. There is already a small number of deprecation messages in the code that do also judge, but we should restrict ourselves as far as possible to providing useful information for developers who find a deprecated entity and wonder what to do. In this case here we should simply state that there is no replacement.